src/Castle.Core/DynamicProxy/ProxyGenerationOptions.cs

```diff
@@ -107,15 +107,7 @@ public void GetObjectData(SerializationInfo info, StreamingContext context)
 
 		public Type BaseTypeForInterfaceProxy { get; set; }
 
-		[Obsolete(
-			"This property is obsolete and will be removed in future versions. Use AdditionalAttributes property instead. " +
-			"You can use AttributeUtil class to simplify creating CustomAttributeBuilder instances for common cases.")]
-		public IList<Attribute> AttributesToAddToGeneratedTypes
```

Comment:
Can you please update the changelog with a breaking change for two of these properties and removing `IAttributeDisassembler`.